# высота над дном
float32 height